packages/builders/scripts/vats/terminate-original-stkATOM-USD-price-feed-governor.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file Terminate v110 zcf-b1-9f877-stkATOM-USD_price_feed-governor.
+ */
+
+import { E } from '@endo/far';
+
+/// <reference types="@agoric/vats/src/core/types-ambient"/>
+
+/**
+ * @param {BootstrapPowers} powers
+ */
+export const terminateOriginalStkATOMPriceFeedGovernor = async ({
+  consume: { board, governedContractKits },
+}) => {
+  const { Fail, quote: q } = assert;
+  const expectedLabel = 'stkATOM-USD_price_feed';
+  const contractInstanceHandle = await E(board).getValue('board052184');
```

Comment:
The PR description shows how board052184 was derived from v110 using, presumably, a mainnet database. Why v110? I don't see any mention of it in #9483.

Reply:
We'll want to test this in a3p and perhaps emerynet and other instagoric networks. I don't think there's any reason to expect the board id to be the same. I suggest threading it thru as a parameter.

For example, look at how `labelList` set to `A3P_INTEGRATION` in [package.json](https://github.com/Agoric/agoric-sdk/blob/master/a3p-integration/proposals/p%3Aupgrade-19/package.json) by passing it to [upgrade-mintHolder.js](https://github.com/Agoric/agoric-sdk/blob/master/packages/builders/scripts/vats/upgrade-mintHolder.js) which threads it thru to [upgrade-mintHolder-proposal.js](https://github.com/Agoric/agoric-sdk/blob/master/packages/vats/src/proposals/upgrade-mintHolder-proposal.js)

In this case, the off-chain builder and the on-chain core-eval code are in 1 file (using dynamic import of `@agoric/deploy-script-support`). I don't think that gets in the way of passing options.

Reply:
> The PR description shows how board052184 was derived from v110 using, presumably, a mainnet database. Why v110? I don't see any mention of it in #9483.

The idea is to build confidence by targeting a small vat in the software upgrade, to be followed by larger ones in later core-evals. I'll update #9483.

Reply:
> The idea is to build confidence by targeting a small vat in the software upgrade, to be followed by larger ones in later core-evals. I'll update #9483.

https://github.com/Agoric/agoric-sdk/issues/9483#issuecomment-2558559464